data/governments.txt

```diff
@@ -1079,12 +1079,12 @@ government "Quarg (Kor Efret)"
 		"Quarg" 1
 		"Quarg (Hai)" 1
 		"Quarg (Gegno)" 1
-		"Kor Efret" .01
+		"Kor Efret" .1
 		"Kor Mereti" -.01
 		"Kor Sestor" -.01
 	"custom penalties for"
 		"Kor Efret"
-			provoke 0.01
+			provoke 0.1
```

Comment:
This means the Quarg care more about the player provoking the Kor Efreti than the Kor Efreti themselves do.

Reply:
How is that the case if the Efret also have a penalty of 0.1 for `provoke`?

Reply:
Oh, I didn't see that part, sorry.

Why does the player need to lose reputation for provoking the Efreti?

Reply:
For the same reasons as the Gegno - when I brought up that interaction with the Gegno, as in the idea that shooting them is enough to upset the very protective Quarg, a lot of creators thought the Efreti should receive the same treatment. While the Quarg don't care about humans shooting each other, or shooting the Hai, etc., they would care a lot more if the player is shooting at a peaceful, struggling species, especially one as limited as the Efreti.